go/spec/spec.go

```diff
@@ -247,6 +250,14 @@ func (sp Spec) createDatabase() datas.Database {
 	switch sp.Protocol {
 	case "http", "https":
 		return datas.NewRemoteDatabase(sp.Href(), sp.Options.Authorization)
+	case "aws":
+		u, _ := url.Parse(sp.Href())
+		parts := strings.SplitN(u.Host, ":", 2) // [table] [, bucket]?
+		sess := session.Must(session.NewSession(aws.NewConfig().WithRegion("us-west-2")))
```

Comment:
Issue filed to think about this. It might be tough to add another optional thing, because I think the syntax might become ambiguous. We can talk about it when we're all togehter later this week. https://github.com/attic-labs/noms/issues/2999